src/tests/users.test.js

```diff
@@ -0,0 +1,76 @@
+import chaiHttp from 'chai-http';
+import { use, request, expect } from 'chai';
+import app from '../app';
+import db from '../models';
+
+use(chaiHttp);
+
+const prefix = '/api/v1';
+describe('/auth/signup', () => {
+  beforeEach(async () => {
+    await db.user.destroy({ where: {}, force: true });
+    await db.user.create({
+      firstName: 'John',
+      lastName: 'Doe',
+      password: '12345678',
+      email: 'john@barefoot.com'
+    });
+  });
```

Comment:
I think you can create a file that can store the mock data for this instead of keeping data in this file